r/R/dplyr-mutate.R

```diff
@@ -151,3 +153,95 @@ ensure_named_exprs <- function(exprs) {
   names(exprs)[unnamed] <- map_chr(exprs[unnamed], format_expr)
   exprs
 }
+
+# Take the input quos and unfold any instances of across()
+# into individual quosures
+unfold_across <- function(.data, quos_in) {
+  quos_out <- list()
+  # Check for any expressions starting with across
+  for (quo_i in seq_along(quos_in)) {
```

Comment:
I don't see any tests where there is more than `mutate(across(...))`, i.e. no other quosures in this list. We should probably test the corners of how across interacts with other mutate expressions.

Reply:
Added now.